qmf/src/plugins/messageservices/imap/imapclient.cpp

```diff
@@ -897,9 +898,17 @@ void ImapClient::commandTransition(ImapCommand command, OperationStatus status)
                 && _pushConnectionsReserved) {
                 _waitingForIdle = true;
                 emit updateStatus( tr("Logging in idle connection" ) );
+#ifdef USE_ACCOUNTS_QT
+                if (_ssoSessionManager) {
+                    ssoProcessLogin();
+                } else {
```

Comment:
why this else?  Shouldn't we force construction of the sso manager here if it doesn't yet exist?  or am I misunderstanding?